server/webapp/WEB-INF/rails.new/webpack/helpers/form_helper.js.msx

```diff
@@ -627,6 +655,18 @@ const f = {
     view (vnode) {
       return (<f.callout type="info" {...vnode.attrs}>{vnode.children}</f.callout>);
     }
+  },
+
+  infoRow: {
```

Comment:
how it is different from `<f.info>`? wrapped in a row? why do we need it?

Reply:
It just wraps the `f.info` in a `f.row`. Did this because it was very common across many pages.

Reply:
then can we edit `f.info` to be wrapped inside a `row`?